tasks/update-sensitive-params.rake

```diff
@@ -1,27 +1,23 @@
 # rebuilds the list of parameters that should be filtered when logging
 task 'update-sensitive-params' do
-  sensitive = []
+  sensitive = Hash.new { |h, k| h[k] = [] }
   BuildTools::Services.each do |svc|
     svc.api['shapes'].each_pair do |shape_name, shape|
       if shape['type'] == 'structure' && shape['members']
         shape['members'].each_pair do |member_name, member_ref|
-          if 
-            member_ref['sensitive'] || 
-            svc.api['shapes'][member_ref['shape']]['sensitive']
-          then
+          if member_ref['sensitive'] || svc.api['shapes'][member_ref['shape']]['sensitive']
             name = AwsSdkCodeGenerator::Underscore.underscore(member_name).to_sym
-            sensitive << name
+            sensitive[svc.name] << name unless sensitive[svc.name].include?(name)
```

Comment:
Rather than using `svc.name` here consider using the serviceId, eg: `svc.api['metadata']['serviceId']`